src/data/navigation/navigation.json

```diff
@@ -228,7 +228,7 @@
         "title": "Release highlights"
       },
       "migration": {
-        "title": "Migration"
+        "title": "Migration guide"
```

Comment:
You might want to call this file "10.x migration guide" to get the URL to include "10" in it. Also, so there's a pattern for the next sibling "11.x migration guide". If somebody is migration from v9 to v11, they'll need to reference both guides.

Reply:
Good idea. Going to go with "v10" to avoid having a period in file name.